charts/pega/README.md

```diff
@@ -857,8 +857,7 @@ pegasearch:
 ```
 ## Deploying Pega with externalized kafka service for stream
 
-Deployment of stream with an externalized Kafka configuration requires Pega Infinity 8.4 or later. Starting in 8.7, Pega deprecated the use of embedded stream (using the Stream tier or “Stream” nodetype).
-For new deployments, Pega recommends deploying Pega Platform using an externalized Kafka configuration as a stream service provider to use your own managed Kafka infrastructure. Pega provides parameters for this configuration in the Pega Helm chart so your stream service starts when your Pega nodes start. For details and requirements about configuring Kafka, see [Kafka Cluster requirements](KafkaClusterRequirement.md).
+Strating from Pega 8.9, all the new deployments are defaulted to Externalized Kafka configuration as a stream provider to use your own managed kafka.
```

Comment:
Beginning with Pega Platform '23, configure the default Pega Helm chart parameters that are appropriate to connect to a Kafka service that you manage in your organization to use as your stream provider for Pega Platform data integrations.